ios/ReactNativePageView.m

```diff
@@ -185,13 +177,6 @@ - (void)setReactViewControllers:(NSInteger)index
 
     NSArray *currentVCs = self.reactPageViewController.viewControllers;
     if (currentVCs.count == 1 && [currentVCs.firstObject isEqual:controller]) {
-        // see: 
```

Comment:
Could you revert that comments too ? It will be much easier to have an information, why this workaround has been implemented